tests/always/test_example_dags.py

```diff
@@ -36,6 +44,29 @@
         "The test is skipped because we are running in limited Pydantic environment", allow_module_level=True
     )
 
+# Some certain of examples/system tests might require additional dependencies,
+# which are not installed into specific CI check
+# Format of dictionary:
+# key: prefix of the file which need to be excluded,
+# values: dictionary with package distributions and optional specifier, e.g. >=2.3.4
+OPTIONAL_PROVIDERS_DEPENDENCIES: dict[str, dict[str, str | None]] = {
+    # Regression of https://github.com/apache/airflow/pull/37524
+    # It loads the module now eagerly instead of lazily
+    "tests/system/providers/common/io/example_file_transfer_local_to_s3.py": {"s3fs": None}
```

Comment:
Let's no do this. This one hides the issue

Reply:
I'm think it depend on how does it long takes to fix it.
If we think that it could be fixed soon, than we fix it first and after that made appropriate changes
If do not know how long does it take we have a to option

Option 1: Revert https://github.com/apache/airflow/pull/37524, otherwise we would have an error in the other tests
Option 2: Hide issue (merge this PR as is), create a task and remove it during the fixing

WDYT?

Reply:
Seems like it might be a quick fix

Reply:
https://github.com/apache/airflow/pull/37559

Reply:
But yeah. This fix either not work well. It fix on one case, but not in the others